### DISCUSSION TITLE
使用Windows系统Anaconda环境搭载的Nonebot2机器人如何实现后台运行

### DISCUSSION THREAD

**Discussion by GFCDIAO** [nonebot/discussions#241] [问答] [answered]
如题，请问windows系统的Nonebot2能够实现像Iinux nohup一样的后台运行功能吗？每一次关闭终端或者关闭计算机，机器人也会关闭，需要手动打开。

**Comment by yanyongyu** [comment, marked as answer]
建议租一台云服务器（

**Reply by GFCDIAO** [reply]
我有一个腾讯云的轻量服务器，也有宝塔终端，想请问大佬有相关的搭建教程么？我重新搭建一个试试。感谢

**Reply by yanyongyu** [reply]
那你可以直接用宝塔supervisor插件运行

**Reply by GFCDIAO** [reply]
大佬可以讲的详细一些吗？是需要将nonebot2整个流程在宝塔终端再运行一遍吗？还是说只需要将本地的Nonebot2目录上传到宝塔

**Reply by yanyongyu** [reply]
当然需要在服务器上安装环境，不过服务器上不用anaconda，用miniconda或者venv就行

**Reply by GFCDIAO** [reply]
好的，感谢大佬解答，我重新安装venv环境试一下
